### DISCUSSION TITLE
Updating Just the Database

### DISCUSSION THREAD

**Discussion by nicklamkey** [pyproj4/pyproj#1417] [General]
Is there a way to update just the database containing EPSG codes in pyproj after pyproj is already installed? I have a newly added EPSG code and just updating through conda doesn't always pull the newest database if the version of pyproj is the newest.

**Comment by jjimenezshaw** [comment]
The epsg database is managed by PROJ itself. Check if it was included in PROJ (that is released every 3 months). You can check the version of the epsg db in proj directly, or find your crs in crs-explorer.proj.org . There you can find a selector for different versions of proj.
